app/src/components/HomePage/InventoryPreferencesCard.tsx

```diff
@@ -0,0 +1,59 @@
+import {
+  Button,
+  Card,
+  HStack,
+  IconButton,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import { IoIosClose } from "react-icons/io";
+import { TFunction } from "i18next";
+import React, { useState } from "react";
+import NextLink from "next/link";
+
+export function InventoryPreferencesCard({
+  t,
+  isPublic,
+}: {
+  t: TFunction;
+  isPublic: boolean;
+}) {
+  const [shouldShowInventoryPreferences, setShouldShowInventoryPreferences] =
+    useState(false); // [ON-1840] hide until the feature is finished - don't forget (&& !isPublic)
```

Comment:
Should this be enabled now? Would make sense before we merge it imo.

Reply:
I left it hidden because the feature is not done and has been deprioritized. the purpose of this PR is for the code not to become stale, but it's not ready for production so I've kept it all hidden.